private/bufpkg/bufplugin/bufpluginconfig/bufpluginconfig.go

```diff
@@ -281,6 +295,15 @@ type ExternalNPMRegistryConfig struct {
 	ImportStyle string `json:"import_style,omitempty" yaml:"import_style,omitempty"`
 }
 
+// ExternalMavenRegistryConfig is the external registry configuration for a Maven plugin.
+type ExternalMavenRegistryConfig struct {
+	Deps []struct {
+		GroupID    string `json:"group_id,omitempty" yaml:"group_id,omitempty"`
+		ArtifactID string `json:"artifact_id,omitempty" yaml:"artifact_id,omitempty"`
+		Version    string `json:"version,omitempty" yaml:"version,omitempty"`
```

Comment:
What do you think about using a single GAV string (which supports things like classifier, optional, type, etc.) instead? This would simplify configuration in `buf.plugin.yaml` and any validation client side and let us support more types of dependencies in the future without requiring a CLI release.

Reply:
that makes sense to me - done in https://github.com/bufbuild/buf/pull/1801/commits/2e0cf94ec4a872892a85cdbe0bd8f62db442ad1a. Assuming we'll do validation server-side.